src/commands/createFunction/scriptSteps/ScriptFunctionCreateStep.ts

```diff
@@ -56,8 +57,26 @@ export class ScriptFunctionCreateStep extends FunctionCreateStepBase<IScriptFunc
 
         const language: ProjectLanguage = nonNullProp(context, 'language');
         const fileName: string | undefined = getScriptFileNameFromLanguage(language);
+
+        await this.installDependencies(language, context.projectPath);
         return fileName ? path.join(functionPath, fileName) : functionJsonPath;
     }
 
+    private async installDependencies(language: string, projectPath: string): Promise<void> {
```

Comment:
Maybe `installDependenciesIfNeeded` instead, since it doesn't always do something?